Automattic/wp-calypso
# Delete UpgradeNudge and UpgradeNudgeExpanded

#### Changes proposed in this Pull Request

* This removes `UpgradeNudge` and `UpgradeNudgeExpanded` in favor of `UpsellNudge` as per #38778 

#### Testing instructions

* Switch to this PR
* Navigate to `/earn/ad-settings/[site]` to make sure there are no errors as a result of removing `UpgradeNudgeExpanded`
* Check the old editor to make sure there are no errors as a result of removing `UpgradeNudge`
* Check `/devdocs` to make sure both Blocks have been removed from the examples
